samples/nrf9160/modem_shell/CMakeLists.txt

```diff
@@ -33,3 +33,9 @@ add_subdirectory_ifdef(CONFIG_MOSH_PPP src/ppp)
 add_subdirectory_ifdef(CONFIG_MOSH_FOTA src/fota)
 add_subdirectory_ifdef(CONFIG_MOSH_REST src/rest)
 add_subdirectory_ifdef(CONFIG_MOSH_CLOUD src/cloud)
+
+if(NOT ENV{PROJECT_NAME})
```

Comment:
I'm wondering why this is needed. For example asset_tracker_v2 doesn't seem to have anything similar, but the application name is still sent to the cloud.

Reply:
Yes. With MoSh this is needed to get other than "PROJECT_NAME" as app name in device information in nrf cloud.